SFDC Servicecloud CE-4078

MadhuriKadam-GSLab opened the pull request:
## Highlights
* Added Churros to Salesforce Service Cloud

## Examples
Added Churros for articles and category API's.
## Closes
* Closes #${issueNumber}

tyler-cloud-elements commented on src/test/elements/sfdcservicecloud/resources-category.js:
```diff
@@ -0,0 +1,24 @@
+      "name": payload.name,
+      "pluralLabel": tools.randomStr()
+    };
+    return cloud.post(test.api, payload)
```
This looks good, but why can't you do the normal suite tests? (i.e. `test.should.supportCruds()`, `test.should.supportPagination()` and `test.should.supportCeqlSearch('id')`)

tyler-cloud-elements commented on src/test/elements/sfdcservicecloud/resources-articles.js:
```diff
@@ -5,17 +5,23 @@ const payload = require('./assets/articles');
-      .then(r => cloud.delete(`${test.api}/${articleId}`));
+      .then(r => cloud.put(`${test.api}/${articleId}/status`, statusPayload))
+      .then(r => cloud.delete(`${test.api}/${articleId}`))
+      .then(r => cloud.withOptions({ qs: { q: "select id, name from contact where name like '%foo' limit 10 offset 0" } }).get(`/hubs/helpdesk/query`));
```
I think we need to do bulk in a different way. This just fires a bulk job but doesn't validate the response or the resulting payload. I don't think we should include this line in the test

MadhuriKadam-GSLab commented:
Hi,
I did the changes as you suggested.Removed /query API and for resources-category, I followed the same way that is used in resources-article.I tried to used test.shouldCurds(),  added the transformation of the same but It giving me so I haven't made any change in that.
